### PR TITLE
stackql v0.3.265 (new formula)

### PR DESCRIPTION
**Pull request by datwiz**
Summary:

- Added formula for `stackql`.
- `stackql` is an application that provides an SQL interface for CRUD on arbitrary resources.
- Further information and detailed documentation available at https://stackql.io

- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/stackql.rb
Comment (SMillerDev) [inline Formula/stackql.rb:18]:
```suggestion
      "-X github.com/stackql/stackql/internal/stackql/cmd.BuildMajorVersion=#{version.major}",
      "-X github.com/stackql/stackql/internal/stackql/cmd.BuildMinorVersion=#{version.minor}",
      "-X github.com/stackql/stackql/internal/stackql/cmd.BuildPatchVersion=#{version.patch}",
```

File: Formula/stackql.rb
Comment (SMillerDev) [inline Formula/stackql.rb:27]: We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

File: Formula/stackql.rb
Comment (chenrui333) [reply Formula/stackql.rb:18]: ➕

File: Formula/stackql.rb
Comment (carlocab) [inline Formula/stackql.rb:6]:
```suggestion
  url "https://github.com/stackql/stackql.git",
      tag:      "v0.3.265",
      revision: "ce58a3b9cd00022da42cb8144922613e9491f2f3"
```
Align the arguments to `url`.

File: Formula/stackql.rb
Comment (carlocab) [inline Formula/stackql.rb:21]:
```suggestion
      "-X stackql/internal/stackql/planbuilder.PlanCacheEnabled=true",
```
The quotes don't seem necessary.

Comment (general-kroll-4-life) [reply Formula/stackql.rb]: @carlocab done cheers

File: Formula/stackql.rb
Comment (general-kroll-4-life) [reply Formula/stackql.rb]: @carlocab I've followed the suggested format changes.  As for tarball, `brew` locally throws errors on all permutations I have tried, so I am sticking with the git object pattern.

File: Formula/stackql.rb
Comment (general-kroll-4-life) [reply Formula/stackql.rb]: @SMillerDev done  no problems.  Have added one more test that will interrogate the local file system for (remotely hosted and downloadable) interface definitions, and, finding none, return a header line.

File: Formula/stackql.rb
Comment (general-kroll-4-life) [reply Formula/stackql.rb]: @SMillerDev done cheers

**Comment by general-kroll-4-life** [comment]
@carlocab @SMillerDev @chenrui333 when you have time can you please do another review of this pull request?  Have followed all the suggestions that worked locally and now all checks pass.  Many thanks

**Review by carlocab — APPROVED** [review]
LGTM. Thanks for your patience! I'll wait a little bit to give others the chance to review this again before merging.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**
